### PR TITLE
Feature/nuc 132 transfer dock layout

### PR DESCRIPTION
**Pull request by osmanertem**
image

Screen Shot 2019-04-30 at 17 36 26

### CODE REVIEW

File: docs/.vuepress/components/DockLayoutLiveDemo.vue [hunk 1]
```diff
@@ -0,0 +1,266 @@
+<template>
+  <StackLayout>
+    <StackLayout  class="demo-panel">
+      <Label text="Add new child" class="demo-panel__title"/>
+      <StackLayout orientation="horizontal" style="margin-bottom:5px">
+        <StackLayout orientation="horizontal" class="editor-param">
+        <Label text="Stretch Last child: "/>
+        <input
```
Comment (ahmetceylan) [inline docs/.vuepress/components/DockLayoutLiveDemo.vue]: please replace inputs with textfield

File: docs/.vuepress/components/DockLayoutLiveDemo.vue [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+            <option>left</option>
+            <option>bottom</option>
+            <option>center</option>
+          </select>
+        </StackLayout>
+      </StackLayout>
+
+      <StackLayout orientation="horizontal" class="editor-param">
+        <StackLayout class="editor-param">
+          <button id="docklayout_addchild_button" @click="addChildNode">➕Add Child</button>
```
Comment (ahmetceylan) [inline docs/.vuepress/components/DockLayoutLiveDemo.vue]: please replace html buttons with vuexp button component

File: docs/.vuepress/components/DockLayoutLiveDemo.vue [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+
+    <!-- Children list editor -->
+    <StackLayout orientation="vertical" class="demo-panel">
+      <Label :text="'Children list (' + dockChildrenList.length + ')'" class="demo-panel__title"/>
+      <StackLayout orientation="horizontal" v-for="(dockChild, index) in dockChildrenList">
+        <StackLayout orientation="horizontal" class="editor-param">
+          <Label class="editor-param-title" text="T: "/>
+          <TextField
+            :id="'docklayout_childreneditor_name' + index"
+            type="text"
```
Comment (ercang) [inline docs/.vuepress/components/DockLayoutLiveDemo.vue]: type is not required for textfield

Comment (osmanertem) [reply docs/.vuepress/components/DockLayoutLiveDemo.vue]: fixed, thanks

File: docs/.vuepress/components/DockLayoutLiveDemo.vue (hunk 2 above)
Comment (osmanertem) [reply docs/.vuepress/components/DockLayoutLiveDemo.vue]: replaced with vuexp buttons

**Pull request merged**